Library/Homebrew/cmd/test-bot.rb

```diff
@@ -768,6 +768,9 @@ def run
   end
 
   def test_ci_upload(tap)
+    require "formulary"
+    Formulary.class_eval { define_method(:load_formula) { raise "Loaded formula!" } }
```

Comment:
Yeh, I know about subprocess, it's to make sure we don't in the upload job itself.